CoffeeApp.test.js

```diff
@@ -0,0 +1,9 @@
+var LintStream = require('jslint').LintStream;
+var { data } = require('./CoffeeApp');
+
+test('Verifica Starbuksurile apropiate', () => {

```

Comment:
Can you tell me what is considered to be the optimal rate of test code vs production code?

Reply:
I don't quite understand the question.

Reply:
There a couple of [writings](http://wiki.c2.com/?ProductionCodeVsUnitTestsRatio) out there that suggest that the production code to test code ratio should be aroud 1:2, in other words for every line of production code you should have roughly 2 lines of test code.